src/st_widgets.c

```diff
@@ -764,7 +764,7 @@ static void UpdateMonSec(sbe_widget_t *widget)
 
         for (int i = 0; i < MAXPLAYERS; ++i)
         {
-            int result = 0;
+            int result = 0, others = 0;
```

Comment:
This has become larger than the actual monsec widget code. Should I factor it out into a separate function?